src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller_Backspace.cs

```diff
@@ -63,10 +63,11 @@ private void ExecuteBackspaceOrDelete(ITextView textView, Action nextHandler, bo
                     this.TextView.Caret.PositionChanged += OnCaretPositionChanged;
                 }
 
-                var triggerInfo = CompletionTriggerInfo.CreateBackspaceTriggerInfo(deletedChar);
+                var trigger = new BackspaceOrDeleteCharCompletionTrigger(deletedChar, isBackspace: !isDelete, customTags: _completionTriggerTags);
```

Comment:
On the consumption side, we treat them the same in VB. You're right -- that looks a bit weird.